README.md

```diff
@@ -20,10 +22,13 @@ The OAuth2 authentication of the API for transmart-server is managed by [transma
 
 As an user interface for transmart-server there is a frontend application [transmartApp](transmartApp) built as a "web" profiled Grails platform plugin. There is also a modern cohort selector for i2b2 tranSMART named [Glowing Bear](https://github.com/thehyve/glowing-bear) that can used both with transmart-server and transmart-api-server.
 
+The database schema is described in the [data model documentation](docs/data-model.md).
+The schema is created using Liquibase at application startup, when it is configured to do so
+(see [Setup database](docs#2-setup-database)).
+The [transmart-copy](transmart-copy) tool can be used to load data into the database.
 
-Database definitions and installation instructions are in [transmart-data](transmart-data).
+Legacy database definitions and installation instructions are in [transmart-data](transmart-data).
```

Comment:
In the [installation instruction](https://github.com/thehyve/transmart-core/tree/visit-dimension-docs/docs#2-setup-database) this is still described as a main way of setting up the database. On the other hand the installation instruction is in general a bit outdated (still describing the transmart-server as the main app), so updating it can be done as a separate ticket, including also a link to docker installation instruction.

Reply:
Yes indeed, and hopefully together with a cleaned up repository (without legacy components).